Jellyfin.Server/Filters/AdditionalModelFilter.cs

```diff
@@ -36,17 +35,38 @@ public AdditionalModelFilter(IServerConfigurationManager serverConfigurationMana
         /// <inheritdoc />
         public void Apply(OpenApiDocument swaggerDoc, DocumentFilterContext context)
         {
-            context.SchemaGenerator.GenerateSchema(typeof(LibraryUpdateInfo), context.SchemaRepository);
             context.SchemaGenerator.GenerateSchema(typeof(IPlugin), context.SchemaRepository);
-            context.SchemaGenerator.GenerateSchema(typeof(PlayRequest), context.SchemaRepository);
-            context.SchemaGenerator.GenerateSchema(typeof(PlaystateRequest), context.SchemaRepository);
-            context.SchemaGenerator.GenerateSchema(typeof(TimerEventInfo), context.SchemaRepository);
-            context.SchemaGenerator.GenerateSchema(typeof(SendCommand), context.SchemaRepository);
-            context.SchemaGenerator.GenerateSchema(typeof(GeneralCommandType), context.SchemaRepository);
 
-            context.SchemaGenerator.GenerateSchema(typeof(GroupUpdate<object>), context.SchemaRepository);
+            var websocketTypes = typeof(WebSocketMessage).Assembly.GetTypes()
+                .Where(t => t.IsSubclassOf(typeof(WebSocketMessage))
+                            && !t.IsGenericType
+                            && t != typeof(WebSocketMessageInfo))
+                .ToList();
+
+            context.SchemaRepository.AddDefinition(
+                nameof(InboundWebSocketMessage),
+                new OpenApiSchema
+                {
+                    Type = "object",
+                    Description = "Represents the list of possible inbound websocket types",
+                    OneOf = websocketTypes
+                        .Where(t => typeof(IInboundWebSocketMessage).IsAssignableFrom(t))
+                        .Select(t => context.SchemaGenerator.GenerateSchema(t, context.SchemaRepository))
+                        .ToList()
```

Comment:
Should also add this to the JSON output (for both inbound and outbound):

```
        "discriminator": {
          "propertyName": "MessageType"
        },
```

so it would look like this:
```
"InboundWebSocketMessage": {
        "type": "object",
        "oneOf": [
          {
            "$ref": "#/components/schemas/KeepAliveMessage"
          },
          {
            "$ref": "#/components/schemas/ActivityLogEntryStartMessage"
          },
          {
            "$ref": "#/components/schemas/ActivityLogEntryStopMessage"
          },
          {
            "$ref": "#/components/schemas/ScheduledTasksInfoStartMessage"
          },
          {
            "$ref": "#/components/schemas/ScheduledTasksInfoStopMessage"
          },
          {
            "$ref": "#/components/schemas/SessionsStartMessage"
          },
          {
            "$ref": "#/components/schemas/SessionsStopMessage"
          }
        ],
        "discriminator": {
          "propertyName": "MessageType"
        },
        "description": "Represents the list of possible inbound websocket types"
      },
```

as per my previous comment